source/core/Console.cpp

```diff
@@ -602,7 +602,7 @@ namespace GTA
 	{
 		array<Script ^> ^scripts = ScriptDomain::CurrentDomain->RunningScripts;
 
-		if (scripts->Length == 0)
+		if (scripts->Length <= 1)
```

Comment:
@zorg93 I added it again. There is a good reason it is in that array. Makes management a lot nicer. The reason the console is a script is that it simplifies key management and native calling and creating a workaround for that would probably look a lot worse then the 2 class setup we have now.
@JohnnyCrazy You cannot prevent that from happening, as reloading scripts destroys the entire AppDomain with everything in it in order to unload all assemblies.